HVZ/HVZ/api/views.py

```diff
@@ -34,12 +35,18 @@ def json_get_all_emails(request):
 class Mailer(FormView):
     form_class = MailerForm
     template_name = "api/mailer.html"
```

Comment:
Our link to the mailer page currently maps to api before the actual form for emails. Unless we plan to add other forms to the api folder (Ways of adding/removing players from the database and email groups or to view bounced emails), we might want to remove this subgroup.